src/plugins/intel_gpu/src/graph/impls/ocl/ctc_greedy_decoder.cpp

```diff
@@ -27,16 +27,26 @@ struct ctc_greedy_decoder_impl : typed_primitive_impl_ocl<ctc_greedy_decoder> {
         const auto& primitive = impl_param.typed_desc<ctc_greedy_decoder>();
         auto params = get_default_params<kernel_selector::ctc_greedy_decoder_params>(impl_param);
         auto optional_params = get_default_optional_params<kernel_selector::ctc_greedy_decoder_optional_params>(impl_param.get_program());
+        const auto& outputs_num = primitive->input_size() == 3 ? 2 : static_cast<uint32_t>(primitive->output_size());
```

Comment:
Is this meaningful? Any possibility to have another except 1 in static_cast<uint32_t>(primitive->output_size() for op::v0::CTCGreedyDecoder?

Reply:
It was one of testing code, and as you commented, it's not necessary so it will removed.